.changeset/small-turkeys-change.md

```diff
@@ -0,0 +1,6 @@
+---
+"@twilio-paste/form-pill-group": minor
+"@twilio-paste/core": minor
```

Comment:
```suggestion
"@twilio-paste/form-pill-group": patch
"@twilio-paste/core": patch
```
Because we're not adding any new exports here, this can be a patch for both form-pill-group and core